test/rpc/test_rpc_storage.py

```diff
@@ -1,12 +1,12 @@
 """
 Tests RPC storage
 """
-
 from test.rpc.rpc_utils import rpc_call
 
-import pytest
 from starkware.starknet.public.abi import get_storage_var_address
+import pytest
```

Comment:
Yeah, it was my fault probably, as I messed up the last commit (it even has the same message). Sorry!

The problem is that the linter sometimes says that the imports are in the wrong order, but the formatter doesn't order them when ran. Can pylint maybe order them automatically? It probably can also be set up somewhere in the IDE to order them correctly automatically though.

Reply:
Ok, I can merge then.

Changing the import order is not something a linter does, I think. A formatter could do it, but `black` (the formatter used in our repo), does not support it, as it would [change the AST](https://github.com/psf/black/issues/333). That's why I opened this issue two weeks ago: https://github.com/Shard-Labs/starknet-devnet/issues/339